ios/engine/KMEI/KeymanEngine/Classes/Queries/Queries.swift

```diff
@@ -14,15 +14,18 @@ public class Queries {
     case networkError(Error)
     case noData
     case parsingError(Error)
+    case decodingError(String, String)
 
     var localizedDescription: String {
       switch self {
         case .noData:
-          return "no data returned"
-        case .networkError(let baseError):
-          return "network error occurred - \(String(describing: baseError))"
-        case .parsingError(let baseError):
-          return "failure occurred when parsing results - \(String(describing: baseError))"
+          return engineBundle.localizedString(forKey: "error-query-no-data", value: nil, table: nil)
+        case .networkError(_):
+          return engineBundle.localizedString(forKey: "error-query-general", value: nil, table: nil)
+        case .parsingError(_):
+          return engineBundle.localizedString(forKey: "error-query-decoding", value: nil, table: nil)
```

Comment:
```suggestion
          return engineBundle.localizedString(forKey: "error-query-parsing", value: nil, table: nil)
```

Reply:
https://github.com/keymanapp/keyman/blob/8b47386ca05a8e7116f1ddf579ee614920845c66/ios/engine/KMEI/KeymanEngine/en.lproj/Localizable.strings#L55-L56

Reply:
This is intentionally the same as the other message, much for the same reasons as in that other comment.  A user isn't going to care whether or not it's our custom parsing code vs Swift's default decoding functionality.  We care about that, but they don't.